nbproject/private/private.properties

```diff
@@ -1,6 +1,6 @@
-copy.src.files=false
-copy.src.on.open=false
-copy.src.target=
-index.file=index.php
-run.as=LOCAL
-url=http://localhost:8080/blog/
+copy.src.files=false
+copy.src.on.open=false
+copy.src.target=
+index.file=index.php
+run.as=LOCAL
+url=http://localhost/blog2

```

Comment:
This needs to be blog, not blog2 so that other people can open without an issue.